mlflow/tensorflow.py

```diff
@@ -229,21 +239,35 @@ def load_model(model_uri, tf_sess):
     >>>     output_tensors = [tf_graph.get_tensor_by_name(output_signature.name)
     >>>                       for _, output_signature in signature_def.outputs.items()]
     """
-    local_model_path = _download_artifact_from_uri(artifact_uri=model_uri)
-    tf_saved_model_dir, tf_meta_graph_tags, tf_signature_def_key =\
-        _get_and_parse_flavor_configuration(model_path=local_model_path)
-    return _load_tensorflow_saved_model(tf_saved_model_dir=tf_saved_model_dir, tf_sess=tf_sess,
-                                        tf_meta_graph_tags=tf_meta_graph_tags,
-                                        tf_signature_def_key=tf_signature_def_key)
+
+    if LooseVersion(tensorflow.__version__) < LooseVersion('2.0.0'):
```

Comment:
Looks like most of the code (calling `_download_artifact_from_uri `, `_get_and_parse_flavor_configuration`),  in this if/else block is common across TF 1.x & 2.0, maybe we can pull the common code out of the if/else block?